ingest_check_preservica_for_existing_io.tf

```diff
@@ -1,7 +1,41 @@
 locals {
-  ingest_check_preservica_for_existing_io_lambda_name = "${local.environment}-dr2-ingest-check-preservica-for-existing-io"
+  ingest_check_preservica_for_existing_io_lambda_name_old = "${local.environment}-dr2-ingest-check-preservica-for-existing-io"
+  ingest_check_preservica_for_existing_io_lambda_name     = "${local.environment}-dr2-ingest-find-existing-asset"
```

Comment:
Does this need to have the old name?

Reply:
It does at the moment because the old lambda is still used in the step function. Once the code is deployed to the new lambda, we can switch the step function over to using the new lambda and delete the old name.

Reply:
Sorry, I meant the variable name `ingest_check_preservica_for_existing_io_lambda_name ` not the lambda name

Reply:
Ah I see. Yeah, good point, I've renamed it. I'll rename the file when I delete the old lambda.